dataflow/src/main/java/org/checkerframework/dataflow/constantpropagation/ConstantPropagationStore.java

```diff
@@ -168,6 +168,6 @@ public boolean canAlias(JavaExpression a, JavaExpression b) {
   @Override
   @SuppressWarnings("nullness")
   public String visualize(CFGVisualizer<?, ConstantPropagationStore, ?> viz) {
-    return viz.visualizeStoreKeyVal("constant propagation", null);
+    return viz.visualizeStoreKeyVal("constant propagation", toString());
```

Comment:
Before, this method would not give any useful information about the constant propagation store